play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/ResourceMemoryEvaluator.java

```diff
@@ -112,11 +112,23 @@ private static void evaluateInHumanReadableMode(EvaluationSettings settings) {
      */
     static List<MemoryFootprint> evaluateMemoryFootprint(EvaluationSettings evaluationSettings) {
         try (InputPackage inputPackage = InputPackage.open(evaluationSettings.getWatchFacePath())) {
+
+            AndroidManifest manifest = inputPackage.getManifest();
+
             WatchFaceData watchFaceData =
                     WatchFaceData.fromResourcesStream(
                             inputPackage.getWatchFaceFiles(), evaluationSettings);
             if (!evaluationSettings.isHoneyfaceMode()) {
-                validateFormat(watchFaceData, evaluationSettings.getSchemaVersion());
+                String manifestWffVersion = String.valueOf(manifest.getWffVersion());
+                String cliWffVersion = evaluationSettings.getSchemaVersion();
+                if (cliWffVersion != null && !cliWffVersion.equals(manifestWffVersion)) {
```

Comment:
Also check if `evaluationSettings.reportMode()` is false. If that is set, then the output is expected to be json, so don't print this.